### PR TITLE
LNMU@Deepak

### PR DESCRIPTION
**Pull request by DeepakGautam420**
Added Branch

### CODE REVIEW

File: app/Http/Controllers/Backend/BranchController.php [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        // $branchdata=Branch::get();
+        // return view('Backend.branch',compact('branchdata'));
```
Comment (killerheman) [inline app/Http/Controllers/Backend/BranchController.php:86]: Remove the comment

File: app/Http/Controllers/Backend/BranchController.php [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+     * Remove the specified resource from storage.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
+    {
+        // return $id;exit;
+        $del=Branch::find($id);
+        $del->delete();
```
Comment (killerheman) [inline app/Http/Controllers/Backend/BranchController.php:154]: Put this in a if block like
$res = $del->delete();
if($res){
// session flash code
}
else {
//session flash code
}

Comment (DeepakGautam420) [reply app/Http/Controllers/Backend/BranchController.php:154]: All done Sir

File: app/Http/Controllers/Backend/BranchController.php (hunk 1 above)
Comment (DeepakGautam420) [reply app/Http/Controllers/Backend/BranchController.php:86]: All Done Sir

**Review by killerheman — APPROVED** [review]

**Pull request merged**